concepts.wip/boolean-expressions/about.md

```diff
@@ -1,2 +1,76 @@
 # About
 
+In Julia, `true` and `false` are the logical values of the `Bool` type, and can represent a trivial [boolean expression](https://en.wikipedia.org/wiki/Boolean_expression). Since `true` is a logical statement that unambiguously evaluates to `true`. Similarly can be said for `false`.
+
+## Expressions involving [numeric comparisons](https://docs.julialang.org/en/v1/manual/mathematical-operations/#Numeric-Comparisons)
+
+```julia
+# true is numerically equal to 1
+julia> true ≥ 1
+true
+
+# √ can be typed as \sqrt<tab>
+julia> 120 % 11 == √100
+true
```

Comment:
I kinda hate that `√100` is valid 😬

Reply:
Should I change the unicode character to `sqrt(100)`?

Reply:
That's up to you, I'm fine with either even if I don't like the syntax personally

Reply:
I provided the `sqrt(100)` in a comment as an alternate to the unicode expression for the square root.